### PR TITLE
 buzzinga subject 

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by Hamzaelkhatri — COMMENTED** [review]
All is good.

File: subjects/buzzinga/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+)
+
+func main() {
+	fmt.Println(BuzZinga(15))
+}
+```
+And its output :
+
+```go
+$ go run . | cat -e
```
Comment (MSilva95) [inline subjects/buzzinga/README.md:36]: What output is this?
If you pass 15 isn't the output just "BuzZinga" as the number is divisible by 3 and 5?

File: subjects/buzzinga/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# buzzinga
+
+### Instructions
+Write a function named `BuzZinga()` that takes a number as an argument and prints the following:
+
```
Comment (MSilva95) [inline subjects/buzzinga/README.md:5]: You need to say what the number 0 does...

Comment (MSilva95) [reply subjects/buzzinga/README.md:5]: and how negative works

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/buzzinga/README.md (hunk 2 above)
Comment (zainabdnaya) [reply subjects/buzzinga/README.md]: done

File: subjects/buzzinga/README.md (hunk 1 above)
Comment (zainabdnaya) [reply subjects/buzzinga/README.md:44]: I didn't quite understand you.

Comment (MSilva95) [reply subjects/buzzinga/README.md:44]: my question is what is supposed to be the output for the number 15?
In my perspective, as it is divisible by 3 and 5 the output should be just  "BuzZinga"

File: subjects/buzzinga/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# buzzinga
+
+### Instructions
+
+Write a function named `BuzZinga()` that takes a number as an argument, then  loop over it and prints the following:
```
Comment (MSilva95) [inline subjects/buzzinga/README.md:5]: Write a function called `BuzZinga()` that takes a number as an argument, then loops over it and does the following for each iteration:

File: subjects/buzzinga/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/buzzinga/README.md:44]: Its a bit confusing try changing the head wit this>

Write a function called `BuzZinga()` that takes a number as an argument, then loops over it and does the following for each iteration:

Comment (zainabdnaya) [reply subjects/buzzinga/README.md:44]: Done

File: subjects/buzzinga/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+# buzzinga
+
+### Instructions
+
+Write a function called BuzZinga() that takes a number as an argument, then loops over it and does the following for each iteration:
```
Comment (MSilva95) [inline subjects/buzzinga/README.md:5]: Where is the Highlights in the func?

**Review by MSilva95 — CHANGES_REQUESTED** [review]

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**